milabench/multi.py

```diff
@@ -54,37 +57,60 @@ def make_execution_plan(pack, step=0, repeat=1):
     return exec_plan
 
 
+
+
 class MultiPackage:
     def __init__(self, packs):
         self.packs = packs
 
+    def setup_pack(self):
+        pack = list(self.packs.values())[0]
+        setup_pack = pack.copy({
```

Comment:
This is not that great, is there a way to get an almost empty pack for setup purposes ?